mkdocs.yml

```
@@ -30,51 +30,60 @@ theme:
-    - Creating Dandisets and Uploading Data: "13_upload.md"
-    - Validation Levels for NWB Files: "135_validation.md"
-    - Dandiset Metadata: "136_metadata.md"
-    - Publishing Dandisets: "14_publish.md"
-    - Citing a Dandiset: "citing.md"
-    - Debugging: "15_debugging.md"
-    - DANDI CLI and Python API: https://dandi.readthedocs.io
-  - Developer Guide:
-    - Project Structure: "20_project_structure.md"
-    - Models Integration: "21_models_integration.md"
-    - Notes: "40_development.md"
-    - REST API Swagger: https://api.dandiarchive.org/swagger
-    - REST API Redoc: https://api.dandiarchive.org/redoc
-    - Create DANDI Instance:
-      - Overview and Architecture: "59_overview.md"
-      - Initialize Vendor Accounts for DANDI: "60_initialize_vendors.md"
-      - DANDI Authentication: "61_dandi_authentication.md"
-      - DANDI Client: "62_dandi_cli.md"
-      - DANDI Infrastructure: "63_dandi_infrastructure.md"
-      - DANDI Archive: "64_dandi_archive.md"
-      - DANDI Hub: "65_dandi_hub.md"
-    - Contributing Documentation: "100_contribute_docs.md"
-  - DANDI Hub: "50_hub.md"
+  - Introduction: "introduction.md"
+  - Getting Started:
+    - Creating a DANDI Account: "getting-started/creating-account.md"
+    - DANDI Ecosystem: "getting-started/dandi-ecosystem.md"
+    - Data Standards:
+      - Overview: "getting-started/data-standards/index.md"
+      - NWB: "getting-started/data-standards/nwb.md"
+  - "User Guide: Sharing Data":
+    - Creating a Dandiset: "user-guide-sharing/creating-dandiset.md"
+    - Data Licenses: "user-guide-sharing/data-licenses.md"
+    - Converting Data to NWB:
+      - Overview: "user-guide-sharing/converting-data/index.md"
+      - NWB GUIDE: "user-guide-sharing/converting-data/nwb-guide.md"
+      - NeuroConv: "user-guide-sharing/converting-data/neuroconv.md"
+      - PyNWB and MatNWB: "user-guide-sharing/converting-data/pynwb-matnwb.md"
+    - Validating NWB Files: "user-guide-sharing/validating-files.md"
+    - Uploading Data: "user-guide-sharing/uploading-data.md"
+    - Dandiset Metadata: "user-guide-sharing/dandiset-metadata.md"
+    - Publishing Dandisets: "user-guide-sharing/publishing-dandisets.md"
+    - Citing your own Dandiset: "user-guide-sharing/citing-your-dandiset.md"
+    - Contributing an example notebook: "user-guide-sharing/contributing-notebook.md"
+  - "User Guide: Using Data":
+    - Exploring Dandisets: "user-guide-using/exploring-dandisets.md"
+    - Accessing Data:
+      - Overview: "user-guide-using/accessing-data/index.md"
+      - Downloading: "user-guide-using/accessing-data/downloading.md"
+      - Streaming: "user-guide-using/accessing-data/streaming.md"
+      - External Services: "user-guide-using/accessing-data/external-services.md"
+    - Citing Dandisets: "user-guide-using/citing-dandisets.md"
+    - DANDI Hub: "user-guide-using/dandi-hub.md"
   - Tutorials:
     - Dandiset Examples: https://dandi.github.io/example-notebooks
```

Adding these notebooks back.

```suggestion
  - Tutorials:
    - Dandiset Examples: https://dandi.github.io/example-notebooks
    - DANDI User Guide, Part I: "example-notebooks/dandi/DANDI User Guide, Part I.ipynb"
    - DANDI User Guide, Part II: "example-notebooks/dandi/DANDI User Guide, Part II.ipynb"
    - Simple Dandiset Search: "example-notebooks/tutorials/neurodatarehack_2024/simple_dandiset_search.ipynb"
    - Advanced Asset Search: "example-notebooks/tutorials/cosyne_2023/advanced_asset_search.ipynb"
    - Streaming and interacting with NWB data: "example-notebooks/tutorials/bcm_2024/analysis-demo.ipynb"
    - NWB Widget Demo: "example-notebooks/demos/NWBWidget-demo.ipynb"
```